src/clients/BundleDataClient/utils/DataworkerUtils.ts

```diff
@@ -99,6 +100,11 @@ export function prettyPrintV3SpokePoolEvents(
       "destinationChainId",
       (fill) => `${fill.relayer}`
     ),
+    allUnrepayableFillsInRangeByDestinationChainAndRelayer: groupObjectCountsByTwoProps(
+      bundleUnrepayableFillsV3,
+      "repaymentChainId",
+      (fill) => `${fill.relayer}`
```

Comment:
This might be a bit weird post migration, where `relayer` is the address specified in the fill, which is apparently wrong for the resolved repayment chain. So idk how useful this log is going to be vs. the other log that dumps the actual unrepayable fills.

Reply:
good point, wdyt about just deleting invalid fills and unrepayable fills from this log in favor of the subsequent log that dumps all those fills?